tests/api.c

```diff
@@ -23586,6 +23675,133 @@ static void test_wolfssl_PKCS7(void)
 #endif
 }
 
+static void test_wolfSSL_PKCS7_SIGNED_new(void)
+{
+#if defined(OPENSSL_ALL) && defined(HAVE_PKCS7)
+    PKCS7_SIGNED* pkcs7;
+
+    printf(testingFmt, "wolfSSL_PKCS7_SIGNED_new()");
+
+    pkcs7 = wolfSSL_PKCS7_SIGNED_new();
```

Comment:
In these test cases please use the compatibility layer macro. I.E PKCS7_SIGNED_new() instead of wolfSSL_PKCS7_SIGNED_new(). Doing that will test the macro mapping in addition to the function implementation.

Reply:
This goes for all of the new compatibility layer API additions in tests/api.c